pangeo_forge/recipe.py

```diff
@@ -156,6 +160,8 @@ class NetCDFtoZarrSequentialRecipe(BaseRecipe):
     xarray_concat_kwargs: dict = field(default_factory=dict)
     delete_input_encoding: bool = True
     fsspec_open_kwargs: dict = field(default_factory=dict)
+    process_input: Optional[Callable[[xr.Dataset, str], xr.Dataset] = None
```

Comment:
Whoops, sorry my suggestion was missing a `]`.

Might want to make sure `typing.Callable` is imported as well.

Reply:
Thanks @TomAugspurger!